src/main/java/teammates/common/datatransfer/StudentWithOldRegistrationKeyAttributes.java

```diff
@@ -0,0 +1,31 @@
+package teammates.common.datatransfer;
+
+import teammates.storage.entity.CourseStudent;
+import teammates.storage.entity.Student;
+
+public class StudentWithOldRegistrationKeyAttributes extends StudentAttributes {
```

Comment:
I guess it's not really needed, it's to park the logic of converting Student to CourseStudent in a single class so it's more easily removed in future, but I can do it in the constructor instead